docs/documentation/security.rst

```diff
@@ -0,0 +1,1552 @@
+Security
+========
+
+.. contents::
+    :local:
+
+`7.1 Security Overview <#security_overview>`__
+----------------------------------------------
+
+In release 0.9.0.0, the Kafka community added a number of features that,
+used either separately or together, increases security in a Kafka
+cluster. The following security measures are currently supported:
+
+#. Authentication of connections to brokers from clients (producers and
+   consumers), other brokers and tools, using either SSL or SASL. Kafka
+   supports the following SASL mechanisms:
+
+   -  SASL/GSSAPI (Kerberos) - starting at version 0.9.0.0
```

Comment:
We should use macros for version numbers here.

Reply:
@guozhangwang similar to https://github.com/apache/kafka/pull/4536/files#diff-85987f48f1258d9ee486e3191495582dR109 ?

Reply:
I don't think this is correct, these are "starting at version" notes, so an absolute, fixed version is correct here.